mri/reconstruct/gradient.py

```diff
@@ -20,26 +20,26 @@
 
 class GradAnalysis2(GradBasic, PowerMethod):
 
-    def __init__(self, data, fourier_op):
+    def __init__(self, data, fourier_op, max_iter):
```

Comment:
I think `max_iter` in this context is a bit misleading. Ppl might think the gradient is computed iteratively when it's not. You can either rename the variable or add a doc. I think the best way is to rename the variable to something like `max_iter_spec_rad`.

Reply:
Will change to `max_iter_spec_rad`